src/components/settings/SettingsTabs.jsx

```diff
@@ -50,7 +50,7 @@ const SettingsTabs = ({
     const initialTabs = [];
     if (enableLearnerPortal) {
       initialTabs.push(
-        <Tab eventKey={SETTINGS_TABS_VALUES.access} title={SETTINGS_TAB_LABELS.access}>
+        <Tab key={SETTINGS_TABS_VALUES.access} eventKey={SETTINGS_TABS_VALUES.access} title={SETTINGS_TAB_LABELS.access}>
```

Comment:
Prettier helps with these long lines.  Now might be a good opportunity to add it to this repo!

Reply:
I like the idea!